src/components/NewCommentForm.tsx

```diff
@@ -1,8 +1,100 @@
-import React from 'react';
+import React, { useContext, useState } from 'react';
+import cn from 'classnames';
+import { TodosContext } from '../TodoContext';
+import { addComment } from '../utils/api';
+import { Comment } from '../types/Comment';
+
+interface Props {
+  comments: Comment[];
+  setComments: (q: Comment[]) => void;
+}
+
+export const NewCommentForm: React.FC<Props> = ({ comments, setComments }) => {
+  const [commentName, setCommentName] = useState('');
+  const [commentEmail, setCommentEmail] = useState('');
+  const [commentText, setCommentText] = useState('');
+
+  const [isNameError, setIsNameError] = useState(false);
+  const [isEmailError, setIsEmailError] = useState(false);
+  const [isTextError, setIsTextError] = useState(false);
+  const [isResponseError, setIsResponseError] = useState(false);
+
+  const [isLoading, setIsLoading] = useState(false);
+
+  const context = useContext(TodosContext);
+
+  const { selectedPost } = context;
+
+  const handleNameChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setCommentName(event.target.value);
+    setIsNameError(false);
+  };
+
+  const handleEmailChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setCommentEmail(event.target.value);
+    setIsEmailError(false);
+  };
+
+  const handleTextChange = (event: React.ChangeEvent<HTMLTextAreaElement>) => {
+    setCommentText(event.target.value);
+    setIsTextError(false);
+  };
```

Comment:
so, if you make comment and error as one instance you will need only one handler